HarpGuru/OverlayMenu/styles/index.ts

```diff
@@ -0,0 +1,42 @@
+import { State } from 'react-native-gesture-handler'
+import { StyleSheet, ViewStyle } from 'react-native'
+
+import { themeSizes, themeColors } from '../../Styles'
+
+const { 7: variableSize, 8: titleSize } = themeSizes
+
+export const styles = StyleSheet.create({
+  // OverlayMenuContainer
+  overlayMenuContainer: {
+    ...StyleSheet.absoluteFillObject,
+    opacity: 0.5,
+  },
+  // MenuContainer
+  menuContainer: {
+    ...StyleSheet.absoluteFillObject,
+    flexDirection: 'row',
+    backgroundColor: themeColors.inertOutline,
+  },
+  // OptionContainer
+  optionContainer: {
+    flex: 1,
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+  },
+  title: {
+    fontSize: titleSize
+  },
+  option: {
+    fontSize: variableSize
+  }
+})
+
+// Dynamic styles for OptionContainer
+export const getDynamicStyles = (state: State): { readonly activeSwipeStyle: ViewStyle } => {
+  return StyleSheet.create({
+    activeSwipeStyle: {
+      backgroundColor: (state === State.ACTIVE ? themeColors.inertOutline : 'transparent')
+    }
+  })
+}
```

Comment:
All of these styles should be put along with their components

Reply:
It feels like distributing the styles isn't actually very useful at this point. There are so many components which aren't styled in this folder, it may be difficult to find where the style is actually applied. Perhaps what's required is a reorganisation of the module.

Reply:
This module can be reorganised at a later date if required. It's not clear to me what direction to take here and it's sufficiently well organised to be merged as is, so I'm just going to leave it.